boards/nRF52840-DK/src/lib.rs

```diff
@@ -0,0 +1,580 @@
+//! Board support crate for the Nordic nRF52840-DK
+//!
+//! This crate is in early development. Not much to see here, right now.
+#![no_std]
+
+//#![deny(missing_docs)]
+#![deny(warnings)]
+
+pub use cortex_m;
+pub use cortex_m_rt;
+pub use embedded_hal;
+pub use nrf52840_hal as hal;
+
+/// Exports traits that are usually needed when using this crate
+pub mod prelude {
+    pub use nrf52840_hal::prelude::*;
+}
+
+// TODO:
+// pub mod debug;
+
+use nrf52840_hal::{
+    prelude::*,
+    gpio::{
+        p0::{
+            self,
+        },
+        p1,
+        Pin,
+        Floating,
+        Input,
+        Output,
+        PushPull,
+        PullUp,
+        Level,
+    },
+    nrf52840_pac::{
+        self as nrf52,
+        CorePeripherals,
+        Peripherals,
+    },
+    spim::{
+        self,
+        Spim
+    },
+    uarte::{
+        self,
+        Uarte,
+        Parity as UartParity,
+        Baudrate as UartBaudrate,
+    },
+};
+
+/// Provides access to all features of the nRF52840-DK board
+#[allow(non_snake_case)]
+#[allow(non_camel_case_types)]
+pub struct nRF52840DK {
+    /// The nRF52's pins which are not otherwise occupied on the nRF52840-DK
+    pub pins: Pins,
+
+    /// The nRF52840-DK UART which is wired to the virtual COM port
+    pub com: Uarte<nrf52::UARTE0>,
```

Comment:
`COM` is a bit of a windows-ism :) Maybe just `uart` or `uarte`?

Reply:
You are absolutely right. Renamed this to `cdc` as this would be the correct term :)